### ISSUE TITLE
Show your own NFT in “for sale” view

### ISSUE THREAD

**Issue by mariano-aguero** [0xProject/0x-launch-kit-frontend#521]

**Comment by chriskalani** [comment]
@burnsben is there a design for this?

**Comment by burnsben** [comment]
Here's what I've come up with. 
<img width="323" alt="Screen Shot 2019-06-03 at 2 27 42 PM" src="https://user-images.githubusercontent.com/29830192/58835909-1a457580-860c-11e9-913b-9a415a65ef44.png">

Design can be found here: https://www.figma.com/file/RW4gjUf5pfu2ajO1BvVWfi/Launch-Kit-ERC-721?node-id=59%3A0

**Comment by unjapones** [comment]
@chriskalani @burnsben does this issue implies adding to the corresponding listings ... ?
1. only my collectibles that are on sale, or
2. all my collectibles

**Comment by chriskalani** [comment]
Let's put it on all views

**Comment by unjapones** [comment]
@chriskalani wrote:
> Let's put it on all views

:+1:

But there is another not so evident question here. I'll add some context:

Currently, the listings "Recenlty listed" and "Most valued" only show collectibles from **other users** that are one sale.

If we got it right, the issue is also about adding "You own" badge _and also 
> show your collectibles in the lists "Recenlty listed" and "Most valued.

Did we got this last part ok? If the answer is yes, should we show **only** my collectibles that are on sale or **all** my collectibles?
